---
bool nav_success
bool human_success
